assume/units/powerplant.py

```diff
@@ -420,22 +426,16 @@ def calc_marginal_cost_with_partial_eff(
     def calculate_min_max_power(self, start: pd.Timestamp, end: pd.Timestamp):
         base_load = self.outputs["energy"][start:end]
         # check if min_power is a series or a float
-        min_power = (
-            self.min_power[start:end].max()
-            if type(self.min_power) is pd.Series
-            else self.min_power
-        )
-        min_delta = min_power - base_load.min()
+        min_power = max(self.min_power, base_load.min())
 
         # check if max_power is a series or a float
         max_power = (
-            self.capacity_factor[start:end].min() * self.max_power
-            if type(self.capacity_factor) is pd.Series
-            else self.capacity_factor * self.max_power
+            self.availability[start] * self.max_power
+            if type(self.availability) is dict
+            else self.max_power
         )
-        max_delta = max_power - base_load.max()
 
-        return min_delta, max_delta
+        return min_power, max_power
```

Comment:
I don't understand why you changed to absolute values here again..

What do you expect base_load to be?